app/client/cli/helpers/context.go

```diff
@@ -0,0 +1,21 @@
+package helpers
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+)
+
+const BusCLICtxKey cliContextKey = "bus"
+
+// NOTE: this is required by the linter, otherwise a simple string constant would have been enough
+type cliContextKey string
+
+func SetValueInCLIContext(cmd *cobra.Command, key cliContextKey, value any) {
+	cmd.SetContext(context.WithValue(cmd.Context(), key, value))
+}
+
+func GetValueFromCLIContext[T any](cmd *cobra.Command, key cliContextKey) (T, bool) {
+	value, ok := cmd.Context().Value(key).(T)
+	return value, ok
```

Comment:
```suggestion
	return cmd.Context().Value(key).(T)
```

Reply:
I don't think you can return the multiple return args form of a type assertion directly. Same goes for retrieving a value from a map.